### ISSUE TITLE
No mark is set in project

### ISSUE THREAD

**Issue by idr4n** [spywhere/vscode-mark-jump#11]
In Insiders (macOS Sierra), I get a `No mark is set.` message whenever I run the `  markJump.jumpToProjectMarks` or the `markJump.jumpToMarks` at the project level. Only the current opened file marks are listed, but if I run the commands from another file without marks but inside the project, no marks are listed. Is this the normal behavior of the extension?

**Comment by spywhere** [comment]
`markJump.jumpToMarks` will show all marks only in currently opened file while `markJump.jumpToProjectMarks` will show all marks across the projects (it will only search according to specified area in the configurations). If no mark has found in either of the command, that message will be show.

**Comment by idr4n** [comment]
I have default configurations, but despite having marks in files other than the current opened one, the `markJump.jumpToProjectMarks` command doesn't find project marks. Is there any configuration I am missing? What do you mean by "it will only search according to specified area in the configurations"?

**Comment by spywhere** [comment]
There is a configuration for searching scope (such as ignore `node_modules` folder for example) and also `markJump.showProjectMarks` which used for enable/disable the project searching at all.

**Comment by idr4n** [comment]
I have default settings, so `markJump.showProjectMarks` is set to `true`, and so far I have used it only in a Go project and marks are set in `.go` files, not in `node_modules`, so none of the folders inside my project should be ignored according to the settings. However I am still not getting the project marks list when triggering the command from a file inside the project but without marks. Wonder why...

**Comment by spywhere** [comment]
If you write the mark correctly with default settings, you should see a total mark in the status bar (be sure to close all opened editors first). If nothing appear, you might write the mark wrong (refers to the README for the syntax and how to add a new one).

**Comment by idr4n** [comment]
I have tried `// NOTE: Text goes here` and `// MARK: Section name` styles and they work only in the current opened file and I can see a a total mark in the status bar. But if the files with marks are closed, marks are not picked by the commands and total count is not shown in the status bar.

**Comment by spywhere** [comment]
Ah, got it. I can reproduce the issue now. I'll let you know once I fixed it.

**Comment by spywhere** [comment]
A fix has been published in v0.7.1.

**Comment by idr4n** [comment]
The workaround didn't solve the issue. Just added it to my setting but still `markJump.jumpToProjectMarks` command is not showing project marks. Only the `markJump.jumpToMarks` command works for me when triggered from the current open file with marks.

**Comment by spywhere** [comment]
I'd love to see your `markJump.includeFilePattern ` and `markJump.excludeFilePattern` configurations. It would be great if you could create a simple repository to reproduce the issue, so I can try it on my copy.

By the way, there is a separate searching involved in the extension ("file-wide" and "project-wide" searching). To make it more concise, please don't include "file-wide" in the issue as it is a separate searching unless necessary.

**Comment by idr4n** [comment]
Sure, I will do that (create the repository) and let you know later today. Thanks for your efforts!

**Comment by idr4n** [comment]
Sorry for the delay. I just created a repo (https://github.com/duranivan/markjump_test), hope it helps clearing the issue. So basically the command `markJump.jumpToMarks` works in the current opened file, but the command `markJump.jumpToProjectMarks` does not work anywhere in the project, whether a file with marks is opened or not (I get the message `No mark is set`).

These are my current keybindings:

```
  {
    "key": "cmd+'",
    "command": "markJump.jumpToMarks"
  },
  {
    "key": "cmd+shift+'",
    "command": "markJump.jumpToProjectMarks"
  }
```

I don't have user defined settings for `markJump.includeFilePattern` or `markJump.excludeFilePattern`, so they are the default ones, that is, 

```
"markJump.excludeFilePattern": "**/node_modules/**",
"markJump.includeFilePattern": "**/**.*",
```

**Comment by spywhere** [comment]
Ah, got it. It's happen on the Insiders version only. I'll take a look for what I can do. Thanks!

**Comment by spywhere** [comment]
For a quick look, Insiders introduced a workspace feature that might affect how searching works.

**Comment by spywhere** [comment]
A patch has been published in v0.7.2.

**Comment by idr4n** [comment]
Great, it is working now. Thanks!
